Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -161,27 +161,27 @@
+ building_btwk_aquae_sulis_baths_01:0 "Abandoned Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nUnfortunately, the departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms. The continued maintenance of amenities, like this spa, was very much an afterthought. As such, it has since fallen into disuse and decay. Many locals now instead opt for bathing in the natural springs themselves." 
+ building_btwk_aquae_sulis_baths_02:0 "Restored Baths"
+ building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nLeft to ruin since the Empire's departure, the bathing complex has since been completely renovated. Men of all manners of life now gather in the communal hub, like citizens of the Rome of old, with both locals and travellers from further afield imparting their custom to the province - a great boon to its economy to be sure."
 
 
  ###################
  # GLASTONBURY TOR #
  ###################
  building_type_btwk_glastonbury_tor_01:0 "Glastonbury Tor"
- building_type_btwk_glastonbury_tor_01_desc:0 "The pre-Christian history of the tor is quite unknown. Archeological evidence indicates there was no permanent settlement until after around the 6th century AD, and would later give way to Glastonbury Abbey. Arthurian legend, however, suggests this as being the homestead of the mythical druid Merlin..."
+ building_type_btwk_glastonbury_tor_01_desc:0 ""
  building_btwk_glastonbury_tor_01:0 "$building_type_btwk_glastonbury_tor_01$"
  building_btwk_glastonbury_tor_01_desc:0 "$building_type_btwk_glastonbury_tor_01_desc$"
 
  ##################
  # DUMBARTON ROCK #
  ################## 
  building_type_btwk_dumbarton_rock_01:0 "The Rock of Dumbarton"
- building_type_btwk_dumbarton_rock_01_desc:0 "#italic Dumbarton Rock#! is a rocky outcrop sitting on the edge of the River Clyde overlooking the settlement of Dumbarton, providing ample defence for the fort sitting atop it. It is commonly associated with being the primary seat of the medieval Kingdom of Strathclyde and is even rumoured to have hosted the legendary wizard Merlin at one time..."
- building_btwk_dumbarton_rock_01:0 "$building_type_btwk_glastonbury_tor_01$"
- building_btwk_dumbarton_rock_01_desc:0 "$building_type_btwk_glastonbury_tor_01_desc$"
+ building_type_btwk_dumbarton_rock_01_desc:0 "#italic Dumbarton Rock#! is a volcanic plug sitting on the edge of the River Clyde. It overlooks the settlement of Dumbarton and has for centuries provided ample fortification for the settlement nestled within it. So much so in fact, that many attempts to subjugate its people over the past centuries have been totally defeated, thanks to the sheer cliffs surrounding most of its perimeter. [something about owning the fortress being prestiguous, but managing to besiege it successfully even more so.]"   
```

Comment:
"#italic Dumbarton Rock#! is a volcanic plug sitting on the edge of the River Clyde. -> On the volcanic plug of "#italic Dumbarton Rock#! sits a great fortress, the traditional seat of the Kingdom of Strathclyde, or Alt Clut. 
It overlooks the settlement of Dumbarton and has for centuries provided ample fortification for the settlement nestled within it. So much so in fact, that many attempts to subjugate its people over the past centuries have been totally defeated, thanks to the sheer cliffs surrounding most of its perimeter. -> Overlooking the River Clyde, the fortress is largely surrounded by sheer cliffs and is said to be one of the strongest and most impregnable in all of Britannia. Many have tried and failed to take the fortress over the centuries, making it a powerful stronghold for control of the region. (not sure about the second half of the last sentence, feels a bit uninspired)